### PR TITLE
Added issue template.

### PR DESCRIPTION
**Pull request by akallabeth**
To avoid the usual errors in bug reports add a issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Found a bug? - We would like to help you and smash the bug away.
+
+1. Before reporting a bug have a look into our issue tracker to see if the bug was already reported and you can add some additional information.
+1. If it's a new bug - create a new issue.
+ 
+ To save time and help us identify the issue a bug report should at least contain the following:
+
+    *  an useful description of the bug - "It's not working" isn't good enough - you must try harder ;)
+    * the steps to reproduce the bug
```
Comment (hardening) [inline .github/ISSUE_TEMPLATE.md]: extra space

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+iThis is how are pull requests handled by FreeRDP
+1. Every new pull request needs to pass the builder tests at https://ci.freerdp.com
+1. At least 2 people need to review and test a pull request and agree to accept
```
Comment (hardening) [inline .github/PULL_REQUEST_TEMPLATE.md]: I thought 1 review was enough...

**Comment by hardening** [comment]
Not sure it needs some changes but the rendering of the issue template is not that nice.

**Comment by akallabeth** [comment]
@hardening Ok, refactored the whole template, hope the style is more pleasing now :)

**Comment by akallabeth** [comment]
@freerdp-bot test

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+1. Every new pull request needs to build and pass the unit tests at https://ci.freerdp.com
+1. At least 1 (better two) people need to review and test a pull request and agree to accept
+
+## Preparations before creating a pull
+* Rebase your branch to current master
+* Try to clean up your commit history, group changes to commits
+* Check your formatting! A _astyle_ script can be found at ```./scripts/format_code.sh```
+* Run a clang scanbuild before and after your changes to avoid introducing new bugs
+* Run your compiler at pedantic level to check for new warnings
+
```
Comment (bmiklautz) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'd leave out the last two points - this is (should/will be part of ci). Maybe just add them as hint.
As addition we should add that a pull request shouldn't contain merges.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+## To ease accepting your contribution
+* Give your pull request a proper name and description so people looking at it have an idea what it is for
```
Comment (bmiklautz) [inline .github/PULL_REQUEST_TEMPLATE.md]: Maybe split that:
* Give the pull request a proper name so people looking at it have an basic idea what it is for
* Add at least a brief description what it does (or should do :) and what it's good for

**Comment by bmiklautz** [comment]
@akallabeth good idea. I thought about that a while ago but just didn't do that. Your PR is a perfect starting point :)

**Comment by hardening** [comment]
After @bmiklautz 's comments I came to something, perhaps we should add that the issue template must be deleted from the issue's report (or we'll get very big reports)

**Comment by bmiklautz** [comment]
@akallabeth I think @hardening s idea is quite good. Would be so kind and update the PR to contain something like that?

**Comment by bmiklautz** [comment]
@freerdp-bot test

**Pull request merged**